src/studio/src/designer/DataModeling/Converter/Xml/XmlSchemaToJsonSchemaConverter.cs

```diff
@@ -1100,71 +1100,75 @@ private static void HandleType(XmlQualifiedName typeName, decimal minOccurs, dec
         {
             array = array || maxOccurs > 1;
 
-            JsonSchemaBuilder typeBuilder = builder;
+            var typeBuilder = builder;
```

Comment:
Yes. Logic stayed the same. I just removed nesting.
Basically typeBuilder is the builder reference in case of  non-array.
For arrays we're building items node.